us/en/skymiles/blocks/accordion/accordion.css

```diff
@@ -0,0 +1,135 @@
+.accordion-container {
+    margin-bottom: 20px;
+}
+
+.text p {
+    font-family: var(--heading-font-family);
+    font-size: 1rem;
+    font-weight: 300;
+    line-height: 2.125rem;
+    color: var(--color-delta-blue);
+    padding: 10px;
+}
+
+.accordion-wrapper {
+    width: 100%;
+    margin-left: 0;
+    margin-right: 0;
+}
+
+.accordion-wrapper .toolbar {
+    text-align: right;
+}
+
+.accordion-wrapper .toolbar .tertiary {
+    appearance: none;
+    border: none;
+    background: none;
+    margin: 0;
+    min-width: 4rem;
+    padding: 0.5rem;
+    text-transform: none;
+    letter-spacing: 0.01rem;
+    font-family: var(--button-font-family);
+    text-decoration: none;
+}
+
+.accordion-wrapper .toolbar .tertiary:hover {
+    text-decoration: underline;
+}
+
+.text[aria-expanded="true"] {
+    display: block;
+}
+
+.text[aria-expanded="false"] {
+    display: none;
+}
+
+.accordion-section {
+    padding-top: 20px;
+    padding-bottom: 20px;
+    border-top: .0625rem solid var(--color-delta-border);
+}
+
+.text {
+    display: none;
+}
+
+.visible button,
+.text[aria-expanded="true"] button {
+    min-width: 20rem;
+    width: 20rem;
+    white-space: normal;
+    margin: 5px;
+}
+
+.header {
+    position: relative;
+    padding: 5px;
+}
+
+.header h3 {
+    max-width: 88%;
+    font-size: 1.6rem;
+}
+
+.header:hover {
+    cursor: pointer;
+}
+
+.header::after {
+    content: '';
+    border: 1px solid var(--color-delta-red); 
+    border-width: 2px 2px 0 0; 
+    transform: rotate(135deg);
+    color: var(--color-delta-red);
+    height: 11px;
+    width: 11px;
+    display: block;
+    position: absolute;
+    right: 20px;
+    top: 40%;
+}
+
+.header[aria-expanded="true"]::after {
+    transform: rotate(315deg);
+    top: 45%;
+}
+
+.visible {
+    display: block;
+    padding-top: 20px;
+}
+
+.bold {
+    font-weight: 1200;
```

Comment:
Does this actually work? I thought 1000 was the max, and most fonts would max out at 700 and then not show a difference (unless they have a "black" variant which is bolder than "bold")

Reply:
this was no longer required based on code optimisations, removed it from css file